game-core/src/main/java/games/strategy/engine/framework/startup/launcher/ServerLauncher.java

```diff
@@ -46,7 +47,10 @@
  * Implementation of {@link ILauncher} for a headed or headless network server game.
  */
 @Log
-public class ServerLauncher extends AbstractLauncher {
+public class ServerLauncher extends AbstractLauncher<Boolean> {
```

Comment:
The `Boolean` return value is a bit odd to see. Notably it's not compatible with the 'cousin' class that returns an `Optional<ServerGame>`. Generally, I'd expect any return instances to be variations on optional game objects.

The boolean in addition I think can be made class internal. The value is obtained from a property check, that same property could be checked in both methods. That would make this an `AbstractLauncher<Void>`.

That might be a good step to take. I think eventually down that road we'd look to the call of `AbstractLauncher` to call different interfaces depending on headless or headed.

Reply:
👍 For the `Void` suggestion